API_request_test.py

```diff
@@ -3,6 +3,40 @@
 
 import requests
 import json
+import os, sys
+import argparse
+
+parser = argparse.ArgumentParser(
+                    prog='BetterWeather',
+                    description='This program is written in Python. Its purpose is to collect weather forecast data\n'
+                    'from a few websites (e.g. Open-Meteo, WeatherAPI) and combine it\n'
```

Comment:
```suggestion
                    'from a few free weather forecast APIs (e.g. Open-Meteo, WeatherAPI) and combine it\n'
```
It's really about the right time to start distinguishing those😉